puppet/zulip_ops/manifests/profile/nagios.pp

```diff
@@ -108,19 +108,11 @@
     '/etc/nagios4/conf.d/contacts_nagios2.cfg',
     '/etc/nagios4/conf.d/hostgroups_nagios2.cfg',
     '/etc/nagios4/conf.d/localhost_nagios2.cfg',
+    '/etc/nagios4/conf.d/zulip_nagios.cfg',
   ]:
     ensure     => absent,
   }
 
-  file { '/etc/nagios4/conf.d/zulip_nagios.cfg':
-    ensure => file,
-    mode   => '0644',
-    owner  => 'root',
-    group  => 'root',
-    source => '/usr/local/share/zulip/integrations/nagios/zulip_nagios.cfg',
```

Comment:
I'm pretty sure this is `zulip/integrations/nagios/` from `python-zulip-api`, presumably manually installed via `pip`.